services/database-analyzer.js

```diff
@@ -37,28 +48,49 @@ function DatabaseAnalyzer(databaseConnection, config) {
           if (foreignKey
             && foreignKey.foreign_table_name
             && foreignKey.column_name && !columnInfo.primaryKey) {
+            let alias = _.camelCase(foreignKey.column_name);
+            if (alias.endsWith('Id') && alias.length > 2) {
+              alias = alias.substring(0, alias.length - 2);
+            } else if (alias.endsWith('Uuid') && alias.length > 4) {
+              alias = alias.substring(0, alias.length - 4);
+            }
             const reference = {
               ref: foreignKey.foreign_table_name,
               foreignKey: foreignKey.column_name,
-              as: `_${foreignKey.column_name}`,
+              as: alias,
             };
 
             if (foreignKey.foreign_column_name !== 'id') {
               reference.targetKey = foreignKey.foreign_column_name;
             }
 
+            log(`  Detected ${chalk.bold('relationship')} ${chalk.magenta(reference.as)}`);
+            log(`    with ${chalk.bold('reference to table')} ${chalk.cyan(reference.ref)}`);
+            log(`    with ${chalk.bold('foreign key')} ${chalk.cyan(reference.foreignKey)}`);
+            if (reference.targetKey) {
+              log(`    with ${chalk.bold('target key')} ${chalk.cyan(reference.targetKey)}`);
+            }
             references.push(reference);
           } else if (type) {
             // NOTICE: If the column is of integer type, named "id" and primary, Sequelize will
             //         handle it automatically without necessary declaration.
             if (!(columnName === 'id' && type === 'INTEGER' && columnInfo.primaryKey)) {
               const field = {
+                cameCaseName: _.camelCase(columnName),
                 name: columnName,
                 type,
                 primaryKey: columnInfo.primaryKey,
                 defaultValue: columnInfo.defaultValue,
               };
 
+              const typeOfField = field.primaryKey ? 'primary key' : 'field';
```

Comment:
Isn't a primary key also a field?
if you want to add the primary key information, it should be additional.